slicedimage/_typeformatting.py

```diff
@@ -1,20 +1,39 @@
 import enum
 
 
-def format_tile_dimensions(tile_dimensions):
+def format_tile_coordinates(tile_dimensions):
     """
     Given a dictionary mapping keys to values, where the keys may either be strings or enums, and
-    the values may either be iterables or not, return a new dictionary with the same contents,
-    except the keys are converted to strings and the values that are iterables are converted to
-    tuples.
+    the values may either be a scalar number or an iterable of two scalar numbers, return a new
```

Comment:
When is the input string vs enum?

Reply:
When we send it a map like `{Coordinates.X: ...}` instead of `{"x": xxx}`